README.md

```diff
@@ -4,15 +4,16 @@ Implementation of the default econda tracking plus ecommerce tracking.
 
 ## Configuration
 
-The extension get the values for clientKey, siteid, langid and countryid from a shop config (Key: `sg_connect_tracking_econda`).
+The extension get the values for clientKey, siteid, langid, countryid and sendParentUid from a shop config (Key: `sg_connect_tracking_econda`).
 
 **Example Config:** 
 ```
 {
     "siteid": "de",
     "clientKey": "1234-acbd-",
     "langid": "de",
-    "countryid": "de"
+    "countryid": "de",
+    "sendParentUid": false
```

Comment:
please add the new config option also here https://wiki.shopgate.guru/display/IN/How+to+use+Shop+Configs

Reply:
done